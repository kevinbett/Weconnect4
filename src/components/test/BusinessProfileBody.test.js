import BusinessProfile from '../BusinessProfileBody'
import { shallow } from "enzyme"

describe('Business profile body',() => { 
    it ('Business profile body renders', () => { 
        expect(shallow(<BusinessProfile/>)).toMatchSnapshot()
    });
} )