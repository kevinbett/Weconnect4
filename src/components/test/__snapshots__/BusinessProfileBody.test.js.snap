// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Business profile body Business profile body renders 1`] = `<Route />`;
